### PR TITLE
Make unreachableOrphan a member method

### PR DESCRIPTION
**Pull request by jamesgua**
to be accessible by extended class

Signed-off-by: Tao Guan <james_mango@yahoo.com>

### CODE REVIEW

**Comment by jamesgua** [comment]
In our project we need to extend method **removeEdge**, however method **unreachableOrphan** is local so I made it a protected member method. And I also found enum Start/EndBlock is declared only but not found anywhere else even in open j9 so I removed it.

File: compiler/infra/OMRCfg.hpp [hunk 1]
```diff
@@ -345,6 +345,18 @@ class CFG
 
    TR::Region& getInternalMemoryRegion();
 
+   //FIXME: These public members should eventually be wrtapped in an interface.
+   int32_t                  _max_edge_freq;
+   int32_t                  _calledFrequency;
+   int32_t                  _initialBlockFrequency;
+   static const int32_t MAX_PROF_EDGE_FREQ=0x3FFE;
+   enum OrphanType
+    {
```
Comment (0xdaryl) [inline compiler/infra/OMRCfg.hpp:354]: Fix brace indenting.

File: compiler/infra/OMRCfg.hpp [hunk 2]
```diff
@@ -345,6 +345,18 @@ class CFG
 
    TR::Region& getInternalMemoryRegion();
 
+   //FIXME: These public members should eventually be wrtapped in an interface.
```
Comment (0xdaryl) [inline compiler/infra/OMRCfg.hpp:348]: Please fix spelling of "wrapped" while you're here.

File: compiler/infra/OMRCfg.cpp [hunk 3]
```diff
@@ -719,16 +719,10 @@ TR::CFGNode *OMR::CFG::removeNode(TR::CFGNode *node)
...
-   {
-   IsParented = 0,
-   IsOrphanedNode,
-   IsOrphanedRegion
-   };
-
 // test if 'to' is an unreachable orphan
 //
-static OrphanType unreachableOrphan(TR::CFG *cfg, TR::CFGNode *from, TR::CFGNode *to)
+TR::CFG::OrphanType
```
Comment (0xdaryl) [inline compiler/infra/OMRCfg.cpp:724]: This enum isn't extended and lives in the OMR namespace.  Shouldn't it be `OMR::CFG::OrphanType` ?

**Comment by 0xdaryl** [comment]
Jenkins build all

**Review by 0xdaryl — APPROVED** [review]

**Pull request merged**